### ISSUE TITLE
CDN build with media color mode

### ISSUE THREAD

**Issue by pspeter3** [twbs/bootstrap#39277]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Provide a CDN build where color mode uses media queries instead of the HTML attribute.

### Motivation and context

This allows automatic dark mode more easily for prototypes and examples which use the CDN builds.

**Comment by coliff** [comment]
You can just use a tiny 1 line JavaScript in the head which does the same thing:

```html
<script>
// Switch to dark mode if user has it set
  if (window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches) {
   document.documentElement.setAttribute("data-bs-theme", "dark");
  }
</script>
```

**Comment by pspeter3** [comment]
I agree but it would be nice to have it work automatically through the media query without the need for JavaScript.

**Comment by mdo** [comment]
We may flip the color mode implementation around in v6 to media queries over data attributes fyi.
